source/features/copy-file.tsx

```diff
@@ -3,18 +3,20 @@ import select from 'select-dom';
 import delegate from 'delegate-it';
 import copyToClipboard from 'copy-text-to-clipboard';
 import features from '../libs/features';
+import {groupSiblings} from '../libs/group-buttons';
 
 function handleClick({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
-	const file = button.closest('.Box');
+	const file = button.closest('.Box, .js-gist-file-update-container');
 	const content = select.all('.blob-code-inner', file!)
 		.map(({innerText: line}) => line === '\n' ? '' : line) // Must be `.innerText`
 		.join('\n');
 	copyToClipboard(content);
 }
 
 function renderButton(): void {
-	for (const blameButton of select.all('[data-hotkey="b"]')) {
-		blameButton
+	const selector = features.isGist() ? '.file-actions .btn' : '[data-hotkey="b"]';
```

Comment:
Can these selectors also be merged?

Reply:
This isn’t resolved yet. By “merging” he means what you did with the previous selector where I suggested to drop isGist

Reply:
Sorry I always forget its parameter can be multiple selectors